internal/network/command.go

```diff
@@ -41,6 +41,7 @@ type humanOut struct {
 	DnsResolution                              string `human:"DNS Resolution,omitempty"`
 	DnsDomain                                  string `human:"DNS Domain,omitempty"`
 	ZonalSubdomains                            string `human:"Zonal Subdomains,omitempty"`
+	IdleSince                                  string `human:"Idle Since,omitempty"`
```

Comment:
Looking at the `confluent flink statement` commands, it's better practice to be doing this.
```suggestion
	IdleSince                                  time.Time `human:"Idle Since,omitempty"`
```